README.md

```diff
@@ -111,19 +111,56 @@ Jitpack.io also allows to build from `fuel` forks. If a fork's username is `$you
 
 Fuel requests can be made on the `Fuel` namespace object, any `FuelManager` or using one of the `String` extension methods. If you specify a callback the call is `async`, if you don't it's `blocking`.
 
+#### Async Usage Example
+
 ```kotlin
-"https://httpbin.org/get"
-  .httpGet()
-  .responseString { request, response, result ->
+import com.github.kittinunf.fuel.httpGet
+import com.github.kittinunf.result.Result;
+
+fun main(args: Array<String>) {
+
+    val httpAsync = "https://httpbin.org/get"
+        .httpGet()
+        .responseString { request, response, result ->
+            when (result) {
+                is Result.Failure -> {
+                    val ex = result.getException()
+                    println(ex)
+                }
+                is Result.Success -> {
+                    val data = result.get()
+                    println(data)
+                }
+            }
+        }
+
+    httpAsync.join()
```

Comment:
[nit] I don't think we have this function 🤣

Reply:
The join() call?  that comes from the CancellableRequest returned by responseString()

Reply:
Oh yeah! I didn't know what I was thinking ... I thought this was the synchronous call!